lib-gdal-preferences-ui/src/main/java/org/esa/s2tbx/dataio/gdal/preferences/GDALOptionsPanelController.java

```diff
@@ -0,0 +1,134 @@
+package org.esa.s2tbx.dataio.gdal.preferences;
+
+import org.netbeans.spi.options.OptionsPanelController;
+import org.openide.util.HelpCtx;
+import org.openide.util.Lookup;
+
+import javax.swing.*;
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+
+/**
+ * GDAL Options Panel Controller for GDAL native library loader.
+ * Used for provide an controller for UI to the strategy with loading GDAL native library.
+ *
+ * @author Adrian Draghici
+ */
+@OptionsPanelController.SubRegistration(
```

Comment:
Can we remove the references to S2TBX here?
This is now a general SNAP GFAL Loader

Reply:
Updates made.